From 08c62f4bd0821510681f6a171828a52a0ee630b3 Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Thu, 5 Jul 2018 14:37:38 +0100
Subject: [PATCH 1775/2313] ASoC: SOF: byt: fix sparse warnings.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
(cherry picked from commit 0290ba7cb3a7ad62fb2952a8ecfa1727dcf53f76)
---
 sound/soc/sof/intel/byt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/sof/intel/byt.c b/sound/soc/sof/intel/byt.c
index 49ec019..9d06dd3 100644
--- a/sound/soc/sof/intel/byt.c
+++ b/sound/soc/sof/intel/byt.c
@@ -140,12 +140,12 @@ static void byt_block_write(struct snd_sof_dev *sdev, u32 offset, void *src,
 	n = size % 4;
 
 	/* __iowrite32_copy use 32bit size values so divide by 4 */
-	__iowrite32_copy((void *)dest, src, m);
+	__iowrite32_copy(dest, src, m);
 
 	if (n) {
 		for (i = 0; i < n; i++)
 			tmp |= (u32)*(src_byte + m * 4 + i) << (i * 8);
-		__iowrite32_copy((void *)(dest + m * 4), &tmp, 1);
+		__iowrite32_copy(dest + m * 4, &tmp, 1);
 	}
 }
 
-- 
2.7.4

